composeApp/src/commonMain/kotlin/sp/bvantur/tasky/home/presentation/HomeViewModel.kt

```diff
@@ -21,4 +37,32 @@ class HomeViewModel(private val repository: HomeRepository) : ViewStateViewModel
             }
         }
     }
+
+    override fun onUserAction(userAction: HomeUserAction) {
+        when (userAction) {
+            HomeUserAction.CreateNewEvent -> {
+                viewModelScope.launch {
+                    // TODO refactor this part
+                    val fromTime: Instant = Clock.System.now()
+                    val toTime = fromTime.plus(30.minutes).toEpochMilliseconds()
+                    emitSingleEvent(
+                        HomeSingleEvent.NavigateToCreateEvent(
+                            fromTime = fromTime.toEpochMilliseconds(),
+                            toTime = toTime
+                        )
+                    )
+                }
+            }
+
+            HomeUserAction.LogoutUser -> {
+                viewModelScope.launch {
+                    repository.logoutUser().onError {
+                        // TODO handle error
+                    }.onSuccess {
+                        emitSingleEvent(HomeSingleEvent.NavigateToLogin)
+                    }
```

Comment:
Based on your question about handling errors from the logout endpoint, do you see any UI updates you want to make when you receive an error with the logout step here? Instead, any errors should be handled in the Repo layer, and the UI should only care about when the Repo has finished execution.